cmd/migration/v0.9.1-v1/mapper/deposit.go

```diff
@@ -0,0 +1,47 @@
+package mapper
+
+import (
+	v091domain "github.com/tdex-network/tdex-daemon/cmd/migration/v0.9.1-v1/v091-domain"
+	"github.com/tdex-network/tdex-daemon/internal/core/domain"
+)
+
+func (m *mapperService) FromV091DepositsToV1Deposits(
+	deposits []*v091domain.Deposit,
+) ([]*domain.Deposit, error) {
+	res := make([]*domain.Deposit, 0, len(deposits))
+	depositsPerTxId := make(map[string][]v091domain.Deposit)
+	for _, v := range deposits {
+		if _, ok := depositsPerTxId[v.TxID]; !ok {
+			depositsPerTxId[v.TxID] = make([]v091domain.Deposit, 0)
+		}
+		depositsPerTxId[v.TxID] = append(depositsPerTxId[v.TxID], *v)
+	}
+
+	for k, v := range depositsPerTxId {
+		amountPerAsset := make(map[string]uint64)
+		for _, vv := range v {
+			if _, ok := amountPerAsset[vv.Asset]; !ok {
+				amountPerAsset[vv.Asset] = vv.Value
+			} else {
+				amountPerAsset[vv.Asset] += vv.Value
+			}
```

Comment:
can be simplified to:
```suggestion
			amountPerAsset[vv.Asset] += vv.Value
```

Reply:
Done